test/release/examples/primers/distributions.chpl

```diff
@@ -240,86 +240,88 @@ verifyID(BA);
 // is replicated onto each locale. (Note: consistency among these
 // array replicands is NOT maintained automatically.)
 //
-// This replication is observable in some cases but not others,
-// as shown below. Note: this behavior may change in the future.
+// This replication is observable in some cases but not others, as
+// shown below.  In places this example uses a utility function,
```

Comment:
We should re-jigger everything in this primer that talks about Replicated. Because really I don't see how the user can observe replication, other than by comparing values on distinct locales. A TODO for the doc-freeze week?